zolex/doctrine-encrypt-bundle/4.0/post-install.txt

```diff
@@ -0,0 +1,7 @@
+<bg=blue;fg=white>                                                          </>
+<bg=blue;fg=white> You now need to uncomment the wanted algorithm in the    </>
```

Comment:
This is not true, right?

It is already selected on (which is good)

Reply:
I've adapted this issue